src/m365/spo/commands/web/web-set.spec.ts

```diff
@@ -444,6 +454,25 @@ describe(commands.WEB_SET, () => {
     await command.action(logger, { options: { url: 'https://contoso.sharepoint.com/sites/team-a', footerEnabled: false } });
   });
 
+  it('enables navAudienceTargetingEnabled', async () => {
+    const postRequestStub = sinon.stub(request, 'patch').callsFake(async (opts) => {
+      if (JSON.stringify(opts.data) === JSON.stringify({
+        NavAudienceTargetingEnabled: true
+      })) {
+        return {};
```

Comment:
Last comment, could we change this to just a check on  the URL like we do normally? Since we check the body in the assert function, this is not needed anymore. Also let's just return` instead of returning an empty object.